docs/decisions/0009-versioning-for-phdi-lib.md

```diff
@@ -0,0 +1,68 @@
+# 9. Formalize software version numbers using Semantic Versioning (SemVer)
+
+Date: 2022-6-28
+
+## Status
+
+Accepted
+
+## Context and Problem Statement
+
+As the PHDI building blocks software continues to develop, documenting the versions that the software goes through will become increasingly important. Users will need to be able to track released versions of the software to ensure that they're using the correct version for their use cases, and so that they can receive proper support in the event they experience a software problem. Developers need to be able to document introductions of new features and bug fixes in existing features. Software versioning allows both groups to avoid confusion around the state of PHDI building blocks library.
```

Comment:
We should elaborate a little more here, pointing out the specific problems Semantic Versioning seems to want to account for, specifically around what the major, minor, and patch versions represent. For example, something like "Users need a mechanism by which to both ensure they're using the appropriate version of the code for their use case as well as identify when updating the library could break their current functionality." would help point out that the purpose behind the major version number is to identify when breaking changes have potentially been introduced.